release-process-documentation/release-team-guides/release-lead.md

```diff
@@ -0,0 +1,209 @@
+# Kubernetes Release Team: Release Team Leader
+
+## Overview:
+
+The release team leader role is responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring a retrospective happens.  The lead is also responsible for ensuring a successor is selected once the release is complete.  
+
+## Authority and Responsibility:
+
+A guiding principle for the lead is be an arbiter of decisions, and not the primary decision-maker.  A lead should constantly search for the best-qualified people or SIGs to guide the decision, not "go it alone" unless it is a very specific concern within the release process itself. When decisions are made they must be weighted in favor of community concerns over those of individuals or specific companies.  Leads must also relinquish any favoritism for the company they work for.  If there is a conflict of interest, the lead must recuse themselves from that decision.  Above all, the release lead is a servant leader to the team and the community.
+
+## Skills and Experience Required:
+
+* This role requires a tremendous amount of experience with the Kubernetes community, code layout, ecosystem projects, organizational norms, governance, SIG structure, architecture, and release process
+
+* To properly serve as a release lead, you must have the following Kubernetes-specific qualifications:
+
+    * If you have never held a release role before, you must shadow the release lead for two full release cycles
+
+    * If you have held a release role, you must shadow the release lead for one full release cycle
+
+    * You must be a member in good standing, and part of the Kubernetes org
```

Comment:
> You must be a member in good standing

Can you formally define this requirement?

Reply:
Wasn't this defined as part of the voting for steering committee nominations?  I was under the impression that there's a canonical list somewhere.

Reply:
Alright, that's fine for now. Needs to be updated in the future is it won't be clear for someone.